docs/README.md

```diff
@@ -172,6 +174,23 @@ CORECLR_PROFILER_PATH_64
 OTEL_DOTNET_AUTO_INTEGRATIONS_FILE
 ```
 
+You can also use the [instrument.sh](../instrument.sh) script which uses following
```

Comment:
You might consider breaking the "Instrument a .NET application" section into two smaller sections: 1) Environment variables and 2) Setup script. Based on my reading, that would help make sure someone who already sets the environment variables at the top doesn't try to run the instrument script when they don't have to

Reply:
Tomorrow I will try to create a separate PR to address it 👍